<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(112, 112, 112);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1161</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPlainTextEdit" name="IsbnManualInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>680</y>
       <width>581</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>690</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ISBN 만가능</string>
     </property>
    </widget>
    <widget class="QLabel" name="bookPhotoLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>251</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255));
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/bookIcon/bookIcon.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="BookTitleLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>170</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>BookTitle</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookIsbnLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>250</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BookIsbn</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookPublisherLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>310</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BookPublisher</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookAuthorLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>360</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BookAuthor</string>
     </property>
    </widget>
    <widget class="QLabel" name="notificationLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>630</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>바코드를 인식시켜주세요. </string>
     </property>
    </widget>
    <widget class="QLabel" name="camLabel">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>470</y>
       <width>291</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/scanClipart/scanClipart.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="manualSearchBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>630</y>
       <width>89</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="videoToTextBtn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>630</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>videoRecgonize</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>570</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>인식이 되지않을경우 수동으로 입력해주세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextPageBtn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>nextPage</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="userListTable">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>701</width>
       <height>511</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>새 열</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>no</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>age</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>나는야</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>나는야</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>나는야</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>너</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>너</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>너</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>그</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>그</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>그</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>녀</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>녀</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>녀</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="previousPageBtn">
     <property name="geometry">
      <rect>
       <x>852</x>
       <y>70</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PreviousPage</string>
     </property>
    </widget>
    <widget class="QLabel" name="userListTitle">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>601</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Adobe Caslon Pro Bold&quot;;</string>
     </property>
     <property name="text">
      <string>ListOutput</string>
     </property>
    </widget>
    <widget class="QPushButton" name="userLIstRefresh">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>140</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>갱신</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
